app/make_figures.jl

```diff
@@ -788,6 +962,124 @@ md"""
 ## Comparing Aletsch runs forward in time using snapshot and time-dependent inputs of basal sliding
 """
 
+# ╔═╡ 1315f741-afa2-4eb5-9ef6-243115ec8746
```

Comment:
@luraess : this is the new fig 7. Maybe you can double check, together with forward_aletsch.jl https://github.com/yiluchen1066/Glaide.jl/pull/7/files#diff-53ecee046e1cdc1f156ba3549f69be8edfbd47f15579fa4770f15d70a2a760bb